client/my-sites/people/main.jsx

```diff
@@ -53,6 +53,15 @@ export default React.createClass( {
 		}
 	},
 
+	renderPeopleSectionNav: function( site ) {
+		const commonProps = omit( this.props, [ 'sites' ] );
+		if ( this.props.filter === 'team' ) {
+			return <PeopleSectionNav { ...commonProps } site={ site } baseUrl={ `/people/team/${this.props.sites.selected}` } />;
```

Comment:
Also, it’s hard for me to see the difference between the two conditions – I needed to look at the code very closely to make find out the difference was only the `baseUrl` component. The semantics and the deeper logic behind the differences wasn’t evident.

For example, if I had to explain it to a person I would say something like: “If we’re filtering by team we would like the search to always reset the filter.”

Since the term “search” doesn’t even appear in this component that the level of abstraction where we’re differentiating is probably too high. If we move this condition to the `PeopleSearch` component it feels like it will be much easier to grasp (and it will have all the information we need – the filter and the current selected site).